### ISSUE TITLE
New-OVIloDirectoryGroup/New-OVServerProfileTemplate empty GroupSID not possible 

### ISSUE THREAD

**Issue by BraendiAT** [HewlettPackard/POSH-HPOneView#588]
When creating a Server Profile Template with managed ILO directory groups, the groupSID field cannot be empty/null when using the functions New-OVIloDirectoryGroup or New-OVServerProfileTemplate. 
But an empty value is valid for this variable. 
See the section of a JSON output of a Server Profile Template
```json
      {
        "settingType": "DirectoryGroups",
        "args": {
          "directoryGroupAccounts": [
            {
              "groupDN": "CN=XX,OU=XX=,DC=xxxx,DC=xxxt",
              **"groupSID": "",**
              "userConfigPriv": false,
              "remoteConsolePriv": true,
              "virtualMediaPriv": false,
              "virtualPowerAndResetPriv": false,
              "iLOConfigPriv": false
            }
          ]
        }
```
OneView 6.30
`$ModuleVersion = '6.00.2719.1668'`

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 6.30.3188.1900](https://github.com/HewlettPackard/POSH-HPEOneView/releases/tag/v6.30.3188.1900) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPEOneView.630/6.30.3188.1900).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
